lib/cmds/start.ts

```diff
@@ -422,6 +438,7 @@ function sendStartedSignal(signal: string, viaIPC: boolean) {
       logger.warn('No IPC channel, sending signal via stdout');
     }
   }
+  console.log(signal);
```

Comment:
Yes.  The signal goes to stdout if it isn't send by and IPC channel.  This functionality was accidentally removed in https://github.com/angular/webdriver-manager/commit/1f9713aff1e7d44de900ed3c74abac532d3e25ff, and I fixed it while debugging this PR.  I could make a issue to document the bug/fix if you'd like